lib/active_model/serializer/associations.rb

```diff
@@ -76,10 +78,6 @@ def has_one(name, options = {})
         def associate(reflection)
           self._reflections = _reflections.dup
```

Comment:
I believe this `dup` is useless. There is no test breaking when I remove it, and I can't think of any case where it would be necessary.